pom.xml

```diff
@@ -2176,6 +2176,7 @@
                 <param>com.puppycrawl.tools.checkstyle.checks.sizes.MethodCountCheckTest</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.whitespace.SingleSpaceSeparatorCheckTest</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.whitespace.EmptyLineSeparatorCheckTest</param>
+                <param>com.puppycrawl.tools.checkstyle.filters.SuppressionXpathFilterTest</param>
```

Comment:
@romani Are you ok to adding this test to `pitest-checkstyle-tree-walker`?

Reply:
This is not ideal, update for TreeWalker is trivial, most likely is covered by one UT. 
@timurt , can you copy test that do coverage for mutations to TreeWalkerTest ?

Reply:
done